sample_consensus/include/pcl/sample_consensus/impl/sac_model_line.hpp

```diff
@@ -97,6 +97,9 @@ pcl::SampleConsensusModelLine<PointT>::computeModelCoefficients (
   model_coefficients[5] = input_->points[samples[1]].z - model_coefficients[2];
 
   model_coefficients.template tail<3> ().normalize ();
+  PCL_DEBUG ("[pcl::SampleConsensusModelLine::computeModelCoefficients] Model is (%g,%g,%g,%g,%g,%g).\n",
```

Comment:
This is repeated in all models. Perhaps add this to the base class and call the base class's function here.

Reply:
What base class function do you mean? `computeModelCoefficients` is pure virtual in sac_model.h

Reply:
Pure virtuals can have an implementation, but you can also add another function, or opt for an intermediate class, etc.

Reply:
But have in mind that the models do not all have the same number of coefficients. If this debug line is put somewhere else, some logic for that would be needed. Overall, I don't really see an advantage in moving this print?

Reply:
No such logic is needed. I didn't look into extracting string from Eigen's `WithFormat` class, but it's possible with https://godbolt.org/z/Wrdn6q

I think my suggestion would also reduce the lines of code needed

Reply:
I am not happy with that solution. The use of `boost::format` and the overall complexity - there may be some fewer lines of code overall, but the function `print_eigen` would have to be in `sac_model.h`, which IMO is ultimately worse. In the future, it might make sense to add macros like `PCL_INFO_STREAM` [like ROS has them](http://wiki.ros.org/roscpp/Overview/Logging), but right now I don't see a problem with the PCL_DEBUGs in the sac_model_*.hpp-files.

Reply:
@SergioRAgostinho PTAL. Apart from this I have no concerns. Rest of the changes LGTM